test/sagemaker_tests/pytorch/training/integration/sagemaker/test_distributed_operations.py

```diff
@@ -114,6 +115,27 @@ def test_dist_operations_fastai_gpu(sagemaker_session, framework_version, ecr_im
     _assert_s3_file_exists(sagemaker_session.boto_region_name, model_s3_url)
 
 
+@pytest.mark.processor("gpu")
+@pytest.mark.integration("pt_s3")
+@pytest.mark.model("resnet18")
+@pytest.mark.skip_cpu
+@pytest.mark.skip_py2_containers
+def test_dist_operations_pt_s3_gpu(sagemaker_session, framework_version, ecr_image):
```

Comment:
nit: Please name this `*s3_plugin_*` to be more specific.

Reply:
Also, please put this in its own test file, because this does not test distributed operations.

Reply:
Added new test file